src/Runner.Common/JobServerQueue.cs

```diff
@@ -66,6 +66,7 @@ public sealed class JobServerQueue : RunnerService, IJobServerQueue
         // common
         private IJobServer _jobServer;
         private IResultsServer _resultsServer;
+        private ILaunchServer _launchServer;
```

Comment:
`_launchServer` should not be initialized as part of `jobserverqueue`, I would put it in `jobrunner.cs` and similar to where we initialize the `JobServer`

Reply:
Isn't this following the exact same pattern as both the `JobServer` and the `ResultsServer`?

Reply:
not really, I think.
`await runServer.ConnectAsync(systemConnection.Url, jobServerCredential);` in `JobRunner.cs`

JobServerQueue.cs is for stuff that is supposed to run background for reporting status/log/etc, any IXXXServer that is used by the job should be initially in the JobRunner.cs